### PR TITLE
adding NIBSC to organisations

### PR DESCRIPTION
**Pull request by nibscles**
Hi guys,
hope I've made the necessary edits to add NIBSC to the list of organisations using and contributing to Nextflow.

### CODE REVIEW

File: nf-core-contributors.yaml [hunk 1]
```diff
@@ -485,3 +485,16 @@ contributors:
...
+      description: >
+        The National Institute for Biological Standards and Control (NIBSC) plays a leading national and international role in assuring the quality of biological medicines and diagnostics.
+      address: Blanche Ln, South Mimms, Potters Bar EN6 3QG, United Kingdom
+      url: https://nibsc.org
+      image_fn: NIBSC_logo_rgb_screen.svg
+      contact: Francesco Lescai
+      contact_email: francesco.lescai@nibsc.org
+      contact_github: nibscles
+      location: [51.688315, -0.240508]
+      twitter: tokybo
```
Comment (MaxUlysse) [inline nf-core-contributors.yaml]: This is supposed to be the institutional twitter.
Does your institute has any?

Comment (nibscles) [inline nf-core-contributors.yaml]:
```suggestion
      twitter: NIBSC_MHRA
```

Comment (nibscles) [reply nf-core-contributors.yaml]: changed :)

**Comment by MaxUlysse** [comment]
@ewels we have a tiny issue here with the logo.
@nibscles only managed to get a high resolution of it, and I think it's unlikely he'll ever get a vectorised version of it.
He's currently trying to make one himself.
Any recommendations?

**Pull request merged**